conf/log4j.properties

```diff
@@ -12,63 +12,121 @@
 # You should have received a copy of the GNU General Public License along with this program.
 # If not, see <https://www.gnu.org/licenses/>.
 # ***** END LICENSE BLOCK *****
+status = error
+dest = err
+name = PropertiesConfig
+packages = com.zimbra.qa.selenium
 
 
 # Set root logger level to INFO and its only appender to A1.
-log4j.rootLogger=INFO, A1, general
 
 # A1 is set to be a ConsoleAppender.
-log4j.appender.A1=org.apache.log4j.ConsoleAppender
+appender.A1.type=org.apache.log4j.ConsoleAppender
+appender.A1.name=console
+appender.A1.layout.type=PatternLayout
+appender.A1.layout.pattern=%-4r [%t] %-5p %c %x - %m%n
 
-# A1 uses PatternLayout.
-log4j.appender.A1.layout=org.apache.log4j.PatternLayout
-log4j.appender.A1.layout.ConversionPattern=%-4r [%t] %-5p %c %x - %m%n
+logger.A1.name = A1Console
+logger.A1.level = info
+logger.A1.additivity = false
+logger.A1.appenderRef.stdout.ref = console
 
 # A2 is set to be a FileAppender.
-log4j.appender.A2=org.apache.log4j.RollingFileAppender
-log4j.appender.A2.File=${outputDirectory}/selenium.txt
-log4j.appender.A2.MaxFileSize=100KB
+appender.A2.type=org.apache.log4j.RollingFileAppender
+appender.A2.name=fileAppender
+appender.A2.fileName=${outputDirectory}/selenium.txt
+appender.A2.policies.type=Policies
+appender.A2.policies.size.type=SizeBasedTriggeringPolicy
+appender.A2.policies.size.size=100KB
+appender.A2.layout.type=PatternLayout
+appender.A2.layout.pattern=%-4r [%t] %-5p %c %x - %m%n
 
-# A2 uses PatternLayout.
-log4j.appender.A2.layout=org.apache.log4j.PatternLayout
-log4j.appender.A2.layout.ConversionPattern=%-4r [%t] %-5p %c %x - %m%n
-
-# Test Case Step tracer
-#log4j.logger.testcase.trace=TRACE, A3
-#log4j.logger.com.zimbra.qa.selenium.framework.util.HarnessException=A3
+logger.A2.name = A2File
+logger.A2.level = info
+logger.A2.additivity = false
+logger.A2.appenderRef.file.ref = fileAppender
 
 # A2 is set to be a FileAppender.
-log4j.appender.A3=org.apache.log4j.FileAppender
-log4j.appender.A3.File=${outputDirectory}/testcase.txt
-log4j.appender.A3.MaxFileSize=100KB
-log4j.appender.A3.Append=false
+appender.A3.type=org.apache.log4j.RollingFileAppender
```

Comment:
Use RollingFile only as org.apache.log4j.RollingFileAppender is belongs to log4j-1.2.16